### ISSUE TITLE
Timing Track Behavior Has Changed in 2023.5... and no longer functions as expected.

### ISSUE THREAD

**Issue by Alkly47** [xLightsSequencer/xLights#4158]
**Describe the bug**
Based on change-logs, Keith made a change to timing tracks and the VU Meter in 2023.5 (— enh (keith) Add a text filter to the VUMeter timing track triggers to selectively react to timing track effects) that has changed how timing tracks are handled.
I create timing tracks externally in Audacity, and import into xLights as a timing track for sequencing. This change has broken a method I have used for years.
The problem appears to occur primarily when timings "overlap" within the same timing track. (For example: Different "notes", with differing start times, but the same end time. (Think a multi-note scale played on a piano, with sustain)
Versions of xLights 2023.4 and earlier behave as expected, triggering a Timing Event on the VU Meter for each timing mark., despite overlap.
Versions 2023.5 and newer are broken and ignore any of the overlapping timing marks,

**To Reproduce**
Open the attached "Test" folder/sequence in differing versions of xLights (2023.4 or older, and 2023.5 or newer).
Observe how the VU meter renders differently

**Expected behavior**
I expected Timing track and VU meter behavior to remain consistent, despite additional Timing label filters being added in 2023.5

**Screenshots**
Example:
![image](https://github.com/smeighan/xLights/assets/109037181/6e8e9b8d-77e6-49b1-aa25-47cccd6a471e)
Timing marks notated with blue dots are the ones that are ignored in xLights versions 2023.5 and newer.

**Versions (please complete the following information):**
 - OS: Win 10
 - xLights version 2023.5 or newer

**Attachments**
"Test" sequence/folder:
[Polyphonic Test.zip](https://github.com/smeighan/xLights/files/13465084/Polyphonic.Test.zip)

**Comment by keithsw1111** [comment]
This is intentional. Timing tracks were never meant to overlap.

**Comment by AzGilrock** [comment]
Yeah I'm sure we prevented the overlaps years ago that wasn't a recent change.  If timing marks overlap it breaks a ton of things in xLights because all our function assume timing marks do not overlap each other.  You need multiple timing track if you need those overlaps.

**Comment by Alkly47** [comment]
@keithsw1111 
I understand that multiple timing tracks are the ideal scenario, however, Audacity creates the files combined/overlapped.

Is there not a way to change the method within xLights to import and create multiple tracks if a singular overlapped track is detrimental to some inner function? Rather than entirely remove the practical method of using VAMP plugins externally?

@AzGilrock The change definitely happened this year with the release of 2023.5...

**Comment by AzGilrock** [comment]
I believe the change you are seeing happened with the change you mentioned but I'm pointing out we did not change effect overlap functionality.  That was not allowed for a long time.  I just looked at that checkin for the VU meter change and it only involved the VU meter effect so its not possible that overlapping effects had anything to do with it.

**Comment by AzGilrock** [comment]
If xLights is allowing timings to come in with overlap then that’s a bug we will fix.   Overlaps can cause crashing

**Comment by Alkly47** [comment]
The Overlapped timing tracks still import and look the same as they always have...

I'm not sure what point you are trying to make.... The VU Meter is the effect in question, and the fact that it no longer uses the timing track like it used to is the purpose of this GitHub Issue.
I totally understand that overlapped timing tracks are messy and far from ideal, but that is what Audacity outputs, and they have always worked up until recently.

If that change was intentional, as @keithsw1111 has stated, I get it.
However, that then changes this issue from a perceived "Bug", to my request for an "Enhancement" to perhaps change the method of import and continue to allow externally created timing tracks to function by importing as separated tracks.
Detecting overlap, and automatically creating a "sub" track during import, to me seems feasible.... Am I wrong?

**Comment by sslupsky** [comment]
I like the idea of separated timing tracks.  I ran into a similar problem importing a MIDI file that was generated by an ADT (automatic drum transcription) tool.  When importing the notes they are shown overlapped on the same track in xLights.  I would prefer a timing track for each individual MIDI note so I have separate timing tracks for the kick, snare, HiHat, etc.

If we could add an option to the timing import and note import dialogs to separate the tracks that would be helpful.
